opaque_keys/edx/keys.py

```diff
@@ -61,6 +61,13 @@ def make_asset_key(self, asset_type, path):  # pragma: no cover
         """
         raise NotImplementedError()
 
+    @abstractmethod
+    def make_course_key_v2(self):  # pragma: no cover
```

Comment:
@cpennington @clintonb just to confirm, I have to remove this method only or do I need to remove the `make_course_run_key` in class `CourseKeyV2` too?